specification/cognitiveservices/data-plane/ContentSafety/preview/2023-10-15-preview/contentsafety.json

```diff
@@ -1153,16 +1153,12 @@
         "acceptSeverityThreshold": {
           "type": "integer",
           "format": "int32",
-          "description": "Severity less than the threshold will be treated as accepted for the category. The value can only be 2, 4, 6 for FourSeverityLevels."
-        },
-        "judgementKind": {
-          "$ref": "#/definitions/JudgmentKind",
-          "description": "The judgement kind to help server make judgement."
+          "description": "Severity less than the threshold will be treated as accepted for the category. The value can only be -1, 2, 4, 6 for FourSeverityLevels. If the threshold is -1, it means all content are accepted."
         }
       },
       "required": [
         "category",
-        "judgementKind"
+        "acceptSeverityThreshold"
```

Comment:
`-1` seems like such a magic number. Why not just leave the field as optional and, if not specified, treat it like you would a value of -1? That's more RESTful.

Reply:
Have removed the -1, and pushed the new changes just as we discussed offline

Reply:
I don't see that reflected in the change you just pushed: https://github.com/Azure/azure-rest-api-specs/pull/25570/commits/314db099dbb3c1a0e053aeec69a378331a78b7c5

Reply:
Hi heaths,I double checked that avaiable judgementKind doesn't exist in model.tsp or 2023-10-15-preview/coontentsafety.json